tests/scripts/deploy_admission_controller.sh

```diff
@@ -34,7 +34,7 @@ export SERVICE_NAME="rook-ceph-admission-controller"
 echo "$BASE_DIR"
 
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/$CERT_VERSION/cert-manager.yaml
-timeout 150 sh -c 'until [ $(kubectl -n cert-manager get pods --field-selector=status.phase=Running|grep -c ^cert-) -eq 3 ]; do sleep 1; done'
+timeout 300 sh -c 'until [ $(kubectl -n cert-manager get pods --field-selector=status.phase=Running|grep -c ^cert-) -eq 3 ]; do sleep 1; done'
```

Comment:
150 seems long enough already. Instead of increasing the timeout, first how about logging info that will help see the status, such as the cert-manager pod status?

Reply:
right, but on the safe side, I'll make it to 180 and also add cert-manager pod status.